src/interfaces/tidebit_defi_background/apply_cfd_order.ts

```diff
@@ -17,9 +17,10 @@ export type ApplyCFDOrderData =
   | IApplyUpdateCFDOrderData
   | IApplyCloseCFDOrderData;
 
+// TODO: IApplyCFDOrder['data'] is `any` type
 export interface IApplyCFDOrder {
   type: ICFDOrderType;
-  data: ApplyCFDOrderData;
+  data: ApplyCFDOrderData; // TODO: separate to 3 types
   signature: string;
 }
```

Comment:
- Separate the `IApplyCreateCFDOrder, update, close` by extending `IApplyCFDOrder`